### PR TITLE
Added option to kick players who do not have the client mod installed.

### LINKED ISSUE #54: feature: kick if not installed
There really needs to be a config option to kick people if they don't have the mod installed. 
With a config to change the kick message.

### PR DESCRIPTION
**Pull request by TheMrEngMan**
This is based on a timeout: if the server does not receive packet(s) from the client mod in time (default within 3 seconds), the player will be kicked.
The kick message includes a link to the Curseforge page.
This is disabled by default.

Closes #54

### CODE REVIEW

**Comment by emielderckx** [comment]
<3

File: src/main/java/su/plo/voice/listeners/PlayerListener.java [hunk 1]
```diff
@@ -75,6 +75,15 @@ public void onPlayerJoin(PlayerJoinEvent e) {
                     String.format("Voice chat is installed but doesn't work. Check if port %d UDP is open.",
                     PlasmoVoice.getInstance().config.port));
         }
+
+        Bukkit.getScheduler().runTaskLater(PlasmoVoice.getInstance(), () -> {
```
Comment (Apehum) [inline src/main/java/su/plo/voice/listeners/PlayerListener.java:79]: Also, It's better to first check if the config has "clientModRequired" set as true, and then do runTaskLater

File: src/main/java/su/plo/voice/listeners/PlayerListener.java [hunk 2]
```diff
@@ -75,6 +75,15 @@ public void onPlayerJoin(PlayerJoinEvent e) {
...
+            if(!SocketServerUDP.clients.containsKey(player)) {
+                if(PlasmoVoice.getInstance().config.clientModRequired) {
+                    PlasmoVoice.logger.info(String.format("Player: %s does not have the mod installed!", player.getName()));
+                    player.kickPlayer(PlasmoVoice.getInstance().getConfig().getString("messages.mod_missing_kick_message"));
```
Comment (Apehum) [inline src/main/java/su/plo/voice/listeners/PlayerListener.java:83]: Use this function instead
```java
PlasmoVoice.getInstance().getMessage("mod_missing_kick_message")
```

**Review by Apehum — CHANGES_REQUESTED** [review]

File: src/main/java/su/plo/voice/listeners/PlayerListener.java (hunk 2 above)
Comment (TheMrEngMan) [reply src/main/java/su/plo/voice/listeners/PlayerListener.java]: Done

File: src/main/java/su/plo/voice/listeners/PlayerListener.java (hunk 1 above)
Comment (TheMrEngMan) [reply src/main/java/su/plo/voice/listeners/PlayerListener.java]: Done

**Review by Apehum — APPROVED** [review]

**Pull request merged**